### ISSUE TITLE
Cannot install mljar-supervised due to numpy version

### ISSUE THREAD

**Issue by zuliani99** [mljar/mljar-supervised#441]
I'm still trying to install mljar with already numpy 1.19.5 installed but I'm getting this error:
```
ERROR: Cannot install -r requirements.txt (line 1), mljar-supervised==0.10.6 and numpy==1.19.5 because these package versions have conflicting dependencies.

The conflict is caused by:
    The user requested numpy==1.19.5
    mljar-supervised 0.10.6 depends on numpy>=1.20.0
    The user requested numpy==1.19.5
    mljar-supervised 0.10.6 depends on numpy>=1.20.0

To fix this you could try to:
1. loosen the range of package versions you've specified
2. remove package versions to allow pip attempt to solve the dependency conflict
```
I can't understand the error because in your requirements.txt the numpy dependency is >= 1.19.5

**Comment by pplonski** [comment]
@zuliani99 maybe try to install directly from github:

```
pip install -q -U git+https://github.com/mljar/mljar-supervised.git@master
```

**Comment by zuliani99** [comment]
Yep, all right. Now I have an other problem still regarding the installation task.
If I want to install either autogluon and mljar-suipervised I'll get lots of dependencies error, I see that in the official website of mljar is provided a benchmark between multiple automl tools among which autogluon. So what version of autogluon and mljar are used in that benchmark?

**Comment by pplonski** [comment]
For comparison the [automlbenchamark](https://github.com/openml/automlbenchmark) was used. The Autogluon results were computed by Autogluon team and are published in their paper. The results from mljar are computed by me, and if I remember correctly the version was ~ `0.9.1`. 

Anyway, for comparisons you can have two different local environments, each for the framework. That's the approach of `automlbenchmark`.
